CameraInspector/PhotoMaker.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include "IFrameHandler.h"
+#include "IFrameSaver.h"
+#include <memory>
+
+namespace CameraInspector
+{
+
+class PhotoMaker : public IFrameHandler
```

Comment:
I guess this class have to be thread-save: from one thread we receive frames from another execute MakePhoto function.